Update Edit-or-cancel-travel-arrangements.md

ryanschaffer opened the pull request:
Updated the instruction for modifying a trip in Expensify travel. the information listed was incorrect. I also have no clue what I am doing. I made some copy changes and added images to the help site. This checklist below seems like it isn't applicable.

ryanschaffer committed:
Update Edit-or-cancel-travel-arrangements.md

Updated the instruction for modifying a trip in Expensify travel. the information listed was incorrect.

OSBotify commented:
A preview of your ExpensifyHelp changes have been deployed to https://8f5407c4.helpdot.pages.dev ⚡️
<!-- Created by actions-cool/maintain-one-comment -->

ryanschaffer committed:
Update Edit-or-cancel-travel-arrangements.md

I put the classic information in the New Expensify section and had deleted the New Expensify selection.

Now there should be Classic info and New Expensify info.

ryanschaffer committed:
Update Edit-or-cancel-travel-arrangements.md

removed a <br>

ryanschaffer commented:
I reviewed the changes and everything looks good!

coleaeason commented on docs/articles/expensify-classic/travel/Edit-or-cancel-travel-arrangements.md:
```diff
@@ -3,9 +3,55 @@ title: Edit or cancel travel arrangements
+1. Navigate to Expensify Travel on an internet browser by clicking **Travel** and then **Book or Manage Travel**
+   ![image](https://github.com/Expensify/App/assets/3697042/c2d0959e-3428-489b-80ee-bfc3477fe96e)
 
+<br>
+<br>
```
can you remove all of these extra line breaks? there's a ton of extra space on the page as a result

ryanschaffer replied:
The spacing looks bad without it

coleaeason reviewed (changes requested):
you also need to make the same changes in `new-expensify/travel/Edit-or-cancel-travel-arrangements.md`

rushatgabhane commented on docs/articles/expensify-classic/travel/Edit-or-cancel-travel-arrangements.md:
```diff
@@ -3,9 +3,55 @@ title: Edit or cancel travel arrangements
+To modify or cancel a travel arrangement,
+<br>
+1. Navigate to Expensify Travel on an internet browser by clicking **Travel** and then **Book or Manage Travel**
+   ![image](https://github.com/Expensify/App/assets/3697042/c2d0959e-3428-489b-80ee-bfc3477fe96e)
```
@ryanschaffer please follow the instructions [here](https://github.com/Expensify/App/tree/main/docs#add-images) to add images.

The URL is incorrect. It should be of this format - `![Image description here]({{site.url}}/assets/images/your_image_name_here.png){:width="100%"}`

```suggestion
   ![image](https://github.com/Expensify/App/assets/3697042/c2d0959e-3428-489b-80ee-bfc3477fe96e)
```
